hosted/gke/helper/helper_cluster.go

```diff
@@ -445,6 +464,28 @@ func CreateGKEClusterOnGCloud(zone string, clusterName string, project string, k
 	return nil
 }
 
+// AddNodePoolOnGCloud adds a nodepool to the GKE cluster via gcloud CLI
+func AddNodePoolOnGCloud(clusterName, zone, project, npName string, extraArgs ...string) error {
```

Comment:
Seems covered in other [PR](https://github.com/rancher/hosted-providers-e2e/pull/102/files#diff-338fccae6e3259d1e4e0a7e58efaa73811e852b2ceeaa2d6d89a986655c71822R483)

Reply:
I have modified the other one to be same as this.